kernel/uGraph.mli

```diff
@@ -50,7 +50,8 @@ type explanation =
   | Path of path_explanation
   | Other of Pp.t
 
-type univ_inconsistency = constraint_type * Sorts.t * Sorts.t * explanation option
+type univ_variable_printers = (Sorts.QVar.t -> Pp.t) * (Level.t -> Pp.t)
```

Comment:
Adding functions on error messages is gonna be problematic for IDEs that may want to seralize this stuff for external comsumption.

Reply:
Ok, so, something still to be done.